src/render/dom/utils/unit-conversion.ts

```diff
@@ -140,10 +148,8 @@ const convertChangedValueTypes = (
         // Restore styles to their **calculated computed style**, not their actual
         // originally set style. This allows us to animate between equivalent pixel units.
         const value = visualElement.getValue(key) as MotionValue
-        setAndResetVelocity(
-            value,
-            positionalValues[key](originBbox, elementComputedStyle)
-        )
+
+        setAndResetVelocity(value, origin[key])
```

Comment:
Can you explain why this would fix it?

Reply:
Yeah its in the comment in the previous line - computed style is live so we need to grab these before we trigger a render from it.

Reply:
So how did this regress then?